pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -50,6 +50,11 @@ const (
 	allowForwarding             = 1
 )
 
+type tapOwner struct {
```

Comment:
The type seems to be generic, not tap specific. Don’t you think?

Reply:
Yes, that analogous to saying you have mammals in a cow farm, not cows.

This is currently exclusively used for creating tap devices. 

Can we let whoever uses this structure in the future for whatever thing they come up w/ renaming to something generic ?

Reply:
I think you confuse usage with definition. The variable/arg name may represent a tap, but its type is generic. With your analogy, any mammal has a name, including a cow. Defining a cow-name type is possible, but strange.

Anyway, it is a nit. I won’t fight over it.